internal/cfgmgr/cfgmgr.go

```diff
@@ -0,0 +1,152 @@
+package cfgmgr
+
+import (
+	"fmt"
+	"strings"
+	"unicode/utf16"
+
+	winio "github.com/Microsoft/go-winio/pkg/guid"
```

Comment:
It's confusing to import this as `winio`, since that name is usually used for the root go-winio package. Can we just keep it as `guid`?